.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "C# (.NET Core)",
+    "build": {
+        "dockerfile": "Dockerfile",
+        "args": {
+            "VARIANT": "3.1",
+            "INSTALL_NODE": "false",
+            "INSTALL_AZURE_CLI": "false"
+        }
+    },
+    "settings": {
+        "terminal.integrated.shell.linux": "/bin/bash"
+    },
+    "extensions": [
+        "ms-dotnettools.csharp"
+    ],
+    "postCreateCommand": "dotnet restore",
+}
```

Comment:
Codacy found an issue: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/gh/System-IO-Abstractions/System.IO.Abstractions/pullRequest?prid=6347840)